docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,616 @@ license: |
   limitations under the License.
 ---
 
-Aggregate functions
\ No newline at end of file
+Spark SQL provides build-in aggregate functions defined in the dataset API and SQL interface. Aggregate functions
+operate on a group of rows and return a single value.
+
+Spark SQL aggregate functions are grouped as <code>agg_funcs</code> in Spark SQL. Below is the list of functions.
```

Comment:
What does this mean? Is this info useful for users?
`Spark SQL aggregate functions are grouped as <code>agg_funcs</code> in Spark SQL.`

Reply:
I see, yah, it is internal. I will remove.